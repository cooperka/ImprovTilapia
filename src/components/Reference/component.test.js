import React from 'react';
import { shallow } from 'enzyme';

import Component from './component';

const props = {
  navigation: {},
  referenceSettings: {
    expandedItems: new Set(),
  },
};

it('renders as expected', async () => {
  const wrapper = shallow(<Component {...props} />);
  expect(wrapper).toMatchSnapshot();
});
