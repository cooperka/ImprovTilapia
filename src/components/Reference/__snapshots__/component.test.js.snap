// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<Component
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Accordion
    expanded={true}
    key="Vocabulary"
    left={[Function]}
    title="Vocabulary"
  >
    <Item
      description="Foo"
      key="Short-form"
      title="Short-form"
    />
    <Item
      description="Bar"
      key="Long-form"
      title="Long-form"
    />
  </Accordion>
</Component>
`;
